# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/base/LayoutConstants.h
# Commit: 62c1b893b282
# Full Hash: 62c1b893b282ecbc628d63fbe6844d68aa621e28
# Author: Alexander Surkov <surkov.alexander@gmail.com>
# Date: 2023-02-09 16:07:01
# Regressor Bug: 1791759
# File Overlap Count: 1
# Description:
#   Bug 1791759 - remove unused ContentRelevancyReason::DescendantOfTopLayerElement r=emilio
#   
#   A small cleanup, follow up from D159693.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D169212
# ==============================================================================

diff -r c4d83a6cced5 -r 62c1b893b282 layout/base/LayoutConstants.h
--- a/layout/base/LayoutConstants.h	Thu Feb 09 01:12:12 2023 +0000
+++ b/layout/base/LayoutConstants.h	Thu Feb 09 01:18:04 2023 +0000
@@ -96,9 +96,6 @@
   // If the content of this Frame is on screen or nearly on screen.
   Visible,
 
-  // If this Frame's element is a descendant of a top layer element.
-  DescendantOfTopLayerElement,
-
   // If this Frame's element has focus in its subtree.
   FocusInSubtree,
 